tests/api/test_payment.py

```diff
@@ -265,6 +264,7 @@ def test_payment_capture_gateway_error(
 """
 
 
+@pytest.mark.xfail
```

Comment:
A reason would be nice :P

Reply:
Right, but I wanted to remove this test at all once I get rid of the "charge" mutation :)